src/components/help/views/ReportSummaryView.tsx

```diff
@@ -38,6 +39,11 @@ export const ReportSummaryView: FC<{}> = props =>
             case ReportType.PHOTO:
                 SendMessageComposer(new CallForHelpFromPhotoMessageComposer(activeReport.extraData, activeReport.cfhTopic, activeReport.roomId, GetSessionDataManager().userId, activeReport.roomObjectId));
                 break;
+            case ReportType.GUIDE:
+                activeReport.reportedChats.forEach(entry => chats.push(entry.webId, entry.message));
```

Comment:
don't need to append any chats to the active report since the report doesn't need any chatlogs